pkg/feeder/feeder.go

```diff
@@ -623,8 +628,7 @@ func (c Client) GetTransactionIDByHash(txHash string) (*string, error) {
 	_, err = c.do(req, &res)
 	if err != nil {
 		metr.IncreaseTxIDFailed()
-		Logger.With("Error", err, "Gateway URL", c.BaseURL).
-			Error("Error connecting to the gateway.")
+		c.logger.Errorw("Error connecting to the gateway.", "error", err, "url", c.BaseURL)
```

Comment:
```suggestion
		c.logger.Errorw("Error connecting to the gateway.", "error", err, "url", c.BaseURL)
```

Reply:
It appears no change is suggested--I tried to remove the rest of the periods from the log messages in a recent commit.